.github/workflows/docker-update.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - uses: elgohr/Publish-Docker-Github-Action@master
+      - uses: elgohr/Publish-Docker-Github-Action@v4
```

Comment:
Is there anything preventing us from going to v5?

Reply:
More information: https://github.com/elgohr/Publish-Docker-Github-Action/releases